NSS-Day-Cohort-69/tabloid-one-more-div
# Added the Category Management view, with all the Categories listed

### **Purpose**
To allow a user with the role admin to see a list of all Categories.

### **Files Changed**
- Created file, and added the class with the needed properties in new ` AllCategoriesDTO.cs`
- Created file, and added the endpoint for getting all Categories in new `CategoryController.cs`
- Created file and added the fetch for getting all Categories in new `CategoryManager.js`
- Created file, and wrote initial rendering and logic for Categories Management view in new `CategoryList.jsx`
- Added routing for Categories Management in view in `ApplicationView.jsx`
- Added nav button to Categories Management view for logged in Admins in `NavBar.jsx`

### **To Test**
Fetch, setup, and run according to project README, then confirm the following

- Logged in Admin can see a nav button for the Categories Management view in the navbar
- Logged in Admin can navigate to the Categories Management view and see a list of all Categories in alphabetical order
- Logged in User with out the Admin role should not be able to see the Categories Management view in the  navbar

closes #38
